packages/kit/test/apps/amp/svelte.config.js

```diff
@@ -15,10 +15,13 @@ const config = {
 				fs: {
 					allow: [path.resolve('../../../src')]
 				},
-
 				// TODO: required to support ipv6, remove on vite 3
 				// https://github.com/vitejs/vite/issues/7075
-				host: 'localhost'
+				host: 'localhost',
+				watch: {
+					// perf, do not watch playwright output dir
+					ignored: ['**/test-results/**']
```

Comment:
Maybe better to just make this part of Kit? Then it'd also work for all our users who use Playwright. Since we offer Playwright support out-of-the-box as an option, I think it would be nice to help our users avoid issues we've hit

Reply:
actually this is something that should be improved in vite itself. filed a bit back https://github.com/vitejs/vite/issues/8341

Reply:
Sent a PR: https://github.com/vitejs/vite/pull/8778

Reply:
merged for Vite 3. I added a TODO to remove it then